learning/web/src/components/Layout/index.js

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 import { Link } from "gatsby"
-import ReactGA from 'react-ga';
-import TagManager from 'react-gtm-module';
+import ReactGA from "react-ga"
+import TagManager from "react-gtm-module"
```

Comment:
Did the previous single quote not work? So it weren't actually importing react-ga or react-gtm-module?

Reply:
These were changes that were made automatically by Prettify (a Visual Studio Code extension - check it out and install it!) when I saved this document. The single quotes did work - either are fine to surround strings in Javascript, they were just out of line with the style convention that we used for this project so they were cleaned up.